pkg/configui/configui.go

```diff
@@ -208,22 +208,10 @@ func BuildScanModeMenu(cfg *config.Instance, pages *tview.Pages, app *tview.Appl
 	return scanMenu
 }
 
-func ConfigUi(cfg *config.Instance, pl platforms.Platform) {
-	app := tview.NewApplication()
-	pages := tview.NewPages()
-
-	tview.Styles.BorderColor = tcell.ColorLightYellow
-	tview.Styles.PrimaryTextColor = tcell.ColorWhite
-	tview.Styles.ContrastSecondaryTextColor = tcell.ColorFuchsia
-	tview.Styles.PrimitiveBackgroundColor = tcell.ColorDarkBlue
-	tview.Styles.ContrastBackgroundColor = tcell.ColorFuchsia
-
-	BuildMainMenu(cfg, pages, app)
-	BuildAudionMenu(cfg, pages, app)
-	BuildReadersMenu(cfg, pages, app)
-	BuildScanModeMenu(cfg, pages, app)
-	pages.SwitchToPage("main")
-
+func misterScreenWorkaround(
```

Comment:
i think we have a duplicate of this somewhere else that can be swapped with this function call